src/components/Week.js

```diff
@@ -7,7 +7,7 @@ import Day from './Day';
 import { daysOfTheWeek } from '../constants/dates';
 import { getActivityItems, getActivitiesForDay } from '../store/activities/selectors';
 
-const Week = ({ activities }) => (
+export const Week = ({ activities }) => (
```

Comment:
If you want to test Unwrapped components (so, without state), I think that more clear is something like that:
before `export default connect(...)`
add line like `export { Week as WeekUnwrapped }`. 
And the, in test file, you can make:
`import { WeekUnwrapped as Week } from ...`